pkg/promotion/promotion.go

```diff
@@ -36,6 +43,21 @@ func BuildsOfficialImages(configSpec *cioperatorapi.ReleaseBuildConfiguration) b
 	return RefersToOfficialImage(promotionName, promotionNamespace)
 }
 
+func isWhiteListed(org, repo string, whitelist WhitelistConfig) bool {
+	if whiteRepos, ok := whitelist.Whitelist[org]; ok {
+		if sets.NewString(whiteRepos...).Has(repo) {
```

Comment:
Uh, converting the list to a set on each call looks wasteful. Can we have a `Load` method that does it once on load time?